js/index.js

```diff
@@ -1,5 +1,38 @@
+function loadData() {
+  const data = document.getElementById('data');
+  if (localStorage.getItem('books')) {
+    data.innerHTML = localStorage.getItem('books');
+  }
+}
 
+function addBook() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const data = document.getElementById('data');
+  const id = new Date().getTime();
+  const book = `
+      <div id="${id}">
+        <p>Title: ${title}<br>Author: ${author}</p>
+        <button type="button" onclick="removeBook(${id})">Remove</button>
+        <hr/>
+      </div>`;
+  data.innerHTML += book;
+  localStorage.setItem('books', data.innerHTML);
```

Comment:
- [ ] Great job for adding the functionality for adding to the localstorage but I highly recommend to use [JSON.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse) – the JSON.parse method converts a string into its corresponding object or value as described by the string, since here the output is like a code in the localstorage but it would be great if you make it like objects ✅🤩💯🚀you can read this [article](https://blog.logrocket.com/storing-retrieving-javascript-objects-localstorage/) to check more about how to save your data in localstorage 🤩

- This is how I could see the data in localstorage in your project
![image](https://user-images.githubusercontent.com/58553711/234011259-eb25c9f4-6460-4764-936e-8dd1d75fd1c7.png)

- It should be somthing like this 
![image](https://user-images.githubusercontent.com/58553711/234011438-30102698-bc01-43e1-aa83-c094dde4d7ed.png)